ddtrace/contrib/tornado/__init__.py

```diff
@@ -3,7 +3,7 @@
 Auto instrumentation is available using the ``patch`` function that **must be called before**
 importing the tornado library.
 
-**Note:** Tornado 5 and 6 supported only for Python 3.7.
+**Note:** Tornado 5 and 6 supports Python 3.7 and above.
```

Comment:
Should we just get rid of this line completely? I don't really know what purpose it's serving

Reply:
Interestingly enough, Tornado 5.X is compatible with Python 2.7 and 3.4+ while Tornado 6.X supports Python 3.5+. I don't see the point of this line unless our integration only supports specific versions of Python (will be testing out to see if we can indeed support 2.7 and 3.4-3.7).

Reply:
Yeah let's only call it out here if this is the case. Right now I think the message is confusing as it sounds like we're stating what versions tornado itself supports

Reply:
The existing note is less confusing but not much better really. Perhaps a better alternative could be:

```
**Note:** This integration requires Python 3.7 and above for Tornado 5 and 6.
```